staging/src/k8s.io/cli-runtime/pkg/genericclioptions/resource/builder.go

```diff
@@ -195,6 +214,9 @@ func (b *Builder) AddError(err error) *Builder {
 // If ContinueOnError() is set prior to this method, objects on the path that are not
 // recognized will be ignored (but logged at V(2)).
 func (b *Builder) FilenameParam(enforceNamespace bool, filenameOptions *FilenameOptions) *Builder {
+	if errs := filenameOptions.validate(); len(errs) > 0 {
+		b.errs = append(b.errs, errs...)
```

Comment:
if this doesn't validate, should we return early?

Reply:
Yes, we should. I'll update it.

Reply:
updated it.